docs/faq.rst

```diff
@@ -1,60 +1,743 @@
 FAQ
-===
+=====
 
-This is the FAQ page. Feel free to contribute any FAQs that you think are missing. 
+This is the FAQ page. Feel free to contribute any FAQs that you think are missing.
```

Comment:
Maybe a TOC element here, in saying that I have not run this up in Sphinx so maybe it does display with one, its been a while since I was using it.